.env

```diff
@@ -1 +1 @@
-ZOOKEEPER_VERSION=3.5.10
\ No newline at end of file
+ZOOKEEPER_VERSION=3.6.4
```

Comment:
Any reason not to do this in a single PR/stage?

Reply:
Updated. Not clear to me if we can jump 2 version at once;

Release notes 3.6: _The upgrade from 3.5.7 to 3.6.0 can be executed as usual, no particular additional upgrade procedure is needed. ZooKeeper 3.6.0 clients are compatible with 3.5 servers as long as you are not using new APIs not present in 3.5._

Release notes 3.7: _The upgrade from 3.6.x to 3.7.0 can be executed as usual, no particular additional upgrade procedure is needed._

Reply:
I see, makes sense then, thanks!